### DISCUSSION TITLE
Lost XCH for offer exchange

### DISCUSSION THREAD

**Discussion by rwal127** [Chia-Network/chia-blockchain#13206] [Support]
Hello! I lost 0.66 XCH in Offer 0xe81bb47e4fec03e8f0d8cd02eac8f96c89626b238d6b72908db00c6c292febd3

"This offer has completed successfully" but no Chia on the wallet. It was 2 days in "pending change". After I click "Actions > Delete unconfirmed transactions" it was lost. How to back it now?

Link to transaction on Dexie [https://dexie.space/offers/8CKcMyDUYBes46wxn3ykVWwK1zRkLQedaTYgKTbziSdT](url)

![image](https://user-images.githubusercontent.com/25417269/187030592-d641c28f-bf2d-4f61-826f-b609a6cc33fb.png)

**Comment by rwal127** [comment]
I have already removed wallet database and waiting for resync if it helps.

**Comment by mohammadroozbahani** [comment]
Hi dear friend
Why are the coins I mined empty from my wallet?
Have I been hacked?

**Comment by mohammadroozbahani** [comment]
![Untitled](https://user-images.githubusercontent.com/112551403/187658131-43d66d58-559a-42c6-bf75-b7ea71bef00c.png)
